src/utils/helpers.ts

```diff
@@ -185,6 +185,33 @@ export const getRegisterWalletWithVerification = ({ parsedRepo, parsedOrg, parse
   }
 };
 
+export const getNewContributorGreeting = ({ parsedRepo, parsedOrg, parsedDefault }: Configs): NewContributorGreeting => {
+  if (parsedRepo && parsedRepo["new-contributor-greeting"] !== undefined && typeof parsedRepo["new-contributor-greeting"] === "object") {
+    let newContributorGreetingConfig = Object(parsedRepo["new-contributor-greeting"]);
```

Comment:
```suggestion
    const newContributorGreetingConfig = parsedRepo["new-contributor-greeting"];
```
you shouldn't use Object() because it removes types and use const if it's not reassigned